compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -824,12 +832,22 @@ void registerCodegenLLVMCPUPasses() {
         buildLLVMCPUVectorLoweringPipeline(funcPassManager, options);
       });
 
-  static PassPipelineRegistration<> LinalgLLVMPipeline(
-      "iree-codegen-linalg-to-llvm-pipeline",
-      "Runs the progressive lowering pipeline from Linalg to LLVM",
-      [](OpPassManager &variantPassManager) {
-        buildLLVMCPUCodegenPassPipeline(variantPassManager);
-      });
+  struct LinalgToLLVMPipelineOptions
+      : public PassPipelineOptions<LinalgToLLVMPipelineOptions> {
+    Option<bool> enableArmSMELoweringPipeline{
+        *this, "enable-arm-sme-lowering-pipeline",
```

Comment:
I think the name could be simplified
```suggestion
        *this, "enable-arm-sme",
```